[2025-03-25T04:27:14.379+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-25T04:27:14.415+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_to_postgresql_2.load_attendance_data manual__2025-03-25T04:25:42.303008+00:00 [queued]>
[2025-03-25T04:27:14.433+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_to_postgresql_2.load_attendance_data manual__2025-03-25T04:25:42.303008+00:00 [queued]>
[2025-03-25T04:27:14.435+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 3
[2025-03-25T04:27:14.489+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): load_attendance_data> on 2025-03-25 04:25:42.303008+00:00
[2025-03-25T04:27:14.511+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=22402) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-25T04:27:14.509+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'api_to_postgresql_2', 'load_attendance_data', 'manual__2025-03-25T04:25:42.303008+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/api_to_pgdatabase.py', '--cfg-path', '/tmp/tmpehv449pf']
[2025-03-25T04:27:14.513+0000] {standard_task_runner.py:72} INFO - Started process 22480 to run task
[2025-03-25T04:27:14.514+0000] {standard_task_runner.py:105} INFO - Job 14: Subtask load_attendance_data
[2025-03-25T04:27:14.669+0000] {task_command.py:467} INFO - Running <TaskInstance: api_to_postgresql_2.load_attendance_data manual__2025-03-25T04:25:42.303008+00:00 [running]> on host a2397d360478
[2025-03-25T04:27:15.025+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='api_to_postgresql_2' AIRFLOW_CTX_TASK_ID='load_attendance_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-25T04:25:42.303008+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-25T04:25:42.303008+00:00'
[2025-03-25T04:27:15.028+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-25T04:27:15.029+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-25T04:27:15.031+0000] {logging_mixin.py:190} INFO - Current task name:load_attendance_data state:running start_date:2025-03-25 04:27:14.415982+00:00
[2025-03-25T04:27:15.033+0000] {logging_mixin.py:190} INFO - Dag name:api_to_postgresql_2 and current dag run status:running
[2025-03-25T04:27:15.034+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-25T04:27:15.082+0000] {base.py:84} INFO - Retrieving connection 'postgres_default_2'
[2025-03-25T04:27:15.368+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-25T04:27:15.399+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-25T04:27:15.401+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=api_to_postgresql_2, task_id=load_attendance_data, run_id=manual__2025-03-25T04:25:42.303008+00:00, execution_date=20250325T042542, start_date=20250325T042714, end_date=20250325T042715
[2025-03-25T04:27:15.436+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-25T04:27:15.438+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-25T04:27:15.439+0000] {logging_mixin.py:190} INFO - Dag name:api_to_postgresql_2 queued_at:2025-03-25 04:25:42.411091+00:00
[2025-03-25T04:27:15.440+0000] {logging_mixin.py:190} INFO - Task hostname:a2397d360478 operator:_PythonDecoratedOperator
[2025-03-25T04:27:15.500+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-25T04:27:15.530+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-25T04:27:15.536+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
